data_prep/introspector.py

```diff
@@ -336,11 +336,20 @@ def _group_function_params(param_types: list[str],
 
 
 def populate_benchmarks_using_introspector(project: str, language: str,
-                                           limit: int, target_oracle: str):
+                                           limit: int,
+                                           target_oracles: List[str]):
   """Populates benchmark YAML files from the data from FuzzIntrospector."""
-  functions = query_introspector_for_targets(project, target_oracle)
+
+  functions = []
+  for target_oracle in target_oracles:
+    logging.info('Extracting functions using oracle %s.', target_oracle)
+    tmp_functions = query_introspector_for_targets(project, target_oracle)
+
+    # Limit the amount of functions from each oracle.
+    functions += tmp_functions[:limit]
+
   if not functions:
-    logging.error('No unreached functions found')
+    logging.error('No functions found using the oracles: {target_oracles}')
```

Comment:
Could you please use lazy format to avoid linter warnings? E.g.,
```python
logging.error('No functions found using the oracles: %s', target_oracles)
```

Reply:
Yes, done